### DISCUSSION TITLE
WW3 mask, bathymetry, and obstruction files for ww3 version 6.07

### DISCUSSION THREAD

**Discussion by JMSnyder1980** [NOAA-EMC/WW3#947] [General]
How do I make obstruction, bathymetry and mask files for the ww3 version 6.07 model.   I want to have the files extend from the prime meridian to the prime meridian again as shown below:
<img width="457" alt="image" src="https://user-images.githubusercontent.com/11950241/225408597-d329e637-edd9-4fc0-aebf-78b38e53a43a.png">
with the latitude extending from -78 to 80 and the spatial resolution 1 degree by 1 degree.  I am trying to do this via Matlab but making the obstruction file takes a very long time over 10 hours.  Is there a way to get this done quickly or are there already files for these made that are available?

**Comment by aliabdolali** [comment]
try gridgen
https://github.com/NOAA-emc/gridgen
it does generate depth, mask and obstr
